spec/services/articles/feeds/variant_query_spec.rb

```diff
@@ -34,6 +34,30 @@
           expect(query_call).to be_a(ActiveRecord::Relation)
           expect(query_call.to_a).not_to match_array(article)
         end
+
+        it "does not return an article published 27 hours before last page view if last comment at is too old" do
+          create_list(:page_view, 5, user: user) # Recent pageviews
```

Comment:
I would probably DRY the `create_list` and `create(:article)`, but that's just nitpicking.